src/StyledComponents/GlobalStyles.js

```diff
@@ -0,0 +1,42 @@
+import styled from 'styled-components';
+
+export const OuterWrapper = styled.section`
+    width: 100%;
+    display: flex;
+    align-items:center;
+    justify-content: center;
+    background-color: ${(props) => (props.grey ? '#F4F4F4;' : 'white')}
+`;
+
+export const InnerWrapper = styled.section`
+    width: 80%;
+    padding-top: 4vh;
+    padding-bottom: 6vh;
+    display: flex; 
+    flex-direction: column;
+
+    @media (min-width: 668px) {
+        width: 60%;
+    }
+
+    @media (min-width: 1025px) {
+        width: 50%;
+    }
+`;
+
+export const P = styled.p`
```

Comment:
Personally I like the more explanatory names, like "const MainText" so it's not misstaken for a "normal" unstyled p-tag. Then if you have several different ones it's easier to set them apart.